mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -367,6 +367,32 @@ class LocalLDAModel private[clustering] (
   }
 
   /**
+   * Predicts the topic mixture distribution for a document (often called "theta" in the
+   * literature).  Returns a vector of zeros for an empty document.
+   *
+   * Note this means to allow quick query for single document. For batch documents, please refer
+   * to [[topicDistributions(documents: RDD[(Long, Vector)])]] to avoid overhead.
+   *
+   * @param document document to predict topic mixture distributions for
+   * @return (document ID, topic mixture distribution for document)
+   */
+  @Since("1.6.0")
+  def topicDistributions(document: Vector): Vector = {
```

Comment:
I just thought of this: Let's rename this ```topicDistribution``` (singular) since it's just computing a single distribution).